src/components/ContactPage/ContactForm.tsx

```diff
@@ -0,0 +1,266 @@
+            <Label htmlFor="name">Hello, my name is:</Label>
+            <FormInput
+              id="name"
+              type="name"
+              placeholder="Name*"
+              required
+              autoComplete="off"
+              {...register('fullName', {
+                required: 'Please enter your name.',
+              })}
+            />
+            {errors.fullName && <Error>{errors.fullName.message}</Error>}
+          </InputWrapper>
+          <InputWrapper>
+            <Label htmlFor="email">My email is:</Label>
+            <FormInput
+              id="email"
+              type="email"
+              placeholder="Email*"
+              required
+              autoComplete="off"
+              {...register('email', {
+                required: 'Please enter your email address.',
+              })}
+            />
+            {errors.email && <Error>{errors.email.message}</Error>}
+          </InputWrapper>
+          <InputWrapper
+            css={css`
+              margin-bottom: ${spacing.lg};
+              min-height: 12.625rem;
+            `}
+          >
+            <Label htmlFor="message">I&apos;d like to talk about:</Label>
+            <FormInput
+              contentEditable
+              as="textarea"
+              id="message"
+              placeholder="Message*"
+              required
+              {...register('message', {
+                required: 'Please include a message.',
+              })}
+            />
+            {errors.message && <Error>{errors.message.message}</Error>}
+          </InputWrapper>
+          <Button type="submit" disabled={isSubmitting}>
+            Submit
+          </Button>
+        </form>
+      )}
+    </>
+  );
+};
+
+export const Contact: FC = () => {
+  return (
+    <ContactWrapper>
+      <ContactAside>
+        <ContactInfo>
+          <Heading
+            variant="h3"
+            color="coral"
+            css={css`
+              margin-bottom: ${cssClamp([0.5, 'mobile'], [2, 'tablet'])};
+            `}
+          >
+            Phone
+          </Heading>
+          <Text variant="h3">614.398.1158</Text>
+        </ContactInfo>
+        <ContactInfo>
+          <Heading
+            variant="h3"
+            color="coral"
+            css={css`
+              margin-bottom: ${cssClamp([0.5, 'mobile'], [2, 'tablet'])};
+            `}
+          >
+            Email
+          </Heading>
+          <Text variant="h3">info@base2.io</Text>
+        </ContactInfo>
+        <ContactInfo>
+          <Heading
+            variant="h3"
+            color="coral"
+            css={css`
+              margin-bottom: ${cssClamp([0.5, 'mobile'], [2, 'tablet'])};
+            `}
+          >
+            Address
+          </Heading>
+          <Text variant="h3">
+            <address
+              css={css`
+                font-style: normal;
+              `}
+            >
+              21 E 5th Ave <br />
+              Suite 102 <br />
```

Comment:
💭 We may not need/want the extra whitespace before the `<br />`s here.

```suggestion
              21 E 5th Ave<br />
              Suite 102<br />
```